src/lms/CurriculumPage.tsx

```diff
@@ -0,0 +1,233 @@
+
+const Container = styled('div', ({ $theme }: { $theme: Theme }) => ({
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  color: $theme.color,
+  backgroundColor: $theme.backgroundColor,
+  minHeight: '100vh',
+}));
+
+const Body = styled('div', {
+  flex: 1,
+  display: 'flex',
+  flexDirection: 'row',
+  '@screen and (max-width: 800px)': {
+    flexDirection: 'column',
+  },
+});
+
+const TopBar = styled('div', ({ $theme }: { $theme: Theme }) => ({
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  borderBottom: `1px solid ${$theme.borderColor}`,
+  height: '48px',
+}));
+
+const StyledTabBar = styled(TabBar, ({ theme }: ThemeProps) => ({
+  flex: 1,
+  borderTopLeftRadius: `${theme.itemPadding * 2}px`,
+  borderTopRightRadius: `${theme.itemPadding * 2}px`,
+  alignSelf: 'end',
+  borderTop: `1px solid ${theme.borderColor}`,
+  borderLeft: `1px solid ${theme.borderColor}`,
+  borderRight: `1px solid ${theme.borderColor}`,
+  backgroundColor: 'white',
+  ':last-child': {
+    marginRight: `${theme.itemPadding * 2}px`,
+  }
+  
+}));
+
+const TopFa = styled(FontAwesome, ({ $theme }: { $theme: Theme }) => ({
+  paddingLeft: `${$theme.itemPadding * 2}px`,
+  paddingRight: `${$theme.itemPadding * 2}px`,
+  fontSize: '32px',
+}));
+
+
+const AddBotballPluginButton = styled('button', ({ $theme }: { $theme: Theme }) => ({
+  // Remove all button styles
+  border: 'none',
+  background: 'none',
+  color: 'inherit',
+  padding: `${$theme.itemPadding * 2}px`,
+  
+  margin: `0 ${$theme.itemPadding * 2}px`,
+  // Add custom styles
+}));
+
+
+const CurriculumPage = ({
+  theme,
+  locale,
+  assignments,
+  myAssignments,
+  onMyAssignmentsChange,
+  userId
+}: CurriculumPagePublicProps & CurriculumPagePrivateProps) => {
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const [isStandardsAligned, setIsStandardsAligned] = React.useState(false);
+  const [subjectsSelected, setSubjectsSelected] = React.useState<Set<Subject>>(new Set());
+  const [standardsSelected, setStandardsSelected] = React.useState<Set<StandardsLocation>>(new Set());
+  const [expandedAssignments, setExpandedAssignments] = React.useState<Set<string>>(new Set());
+  const [gradeLevels, setGradeLevels] = React.useState<[number, number]>([0, 12]);
+  
+  const tabs: TabBar.TabDescription[] = [{
+    name: LocalizedString.lookup(tr('Curriculum'), locale),
+    icon: faSchool
+  }, {
+    name: LocalizedString.lookup(tr('My Assignments'), locale),
+    icon: faSchoolCircleCheck
+  }];
+
+
+  return (
+    <Container $theme={theme}>
+      <TopBar $theme={theme}>
+        <TopFa
+          $theme={theme}
+          icon={faHome}
+          onClick={() => {
+            window.location.pathname = '/';
+          }}
+        />
+
+        <StyledTabBar
+          tabs={tabs}
+          index={tabIndex}
+          onIndexChange={setTabIndex}
+          theme={theme}
+        />
+        {/* <AddBotballPluginButton $theme={theme}>
```

Comment:
Currently the main issue I'm seeing is that there isn't a way to connect the LMS to Google Classroom. It looks like the place where the plugin was intended to be added is commented out.